src/main/java/org/cloudfoundry/promregator/endpoint/MetricsEndpoint.java

```diff
@@ -29,6 +30,9 @@
 public class MetricsEndpoint extends AbstractMetricsEndpoint {
 	private static final Logger log = Logger.getLogger(MetricsEndpoint.class);
 
+	@Value("${promregator.single.endpoint.max:-1}")
```

Comment:
I am not convinced about the name yet. what does "maximal endpoint" mean? How many endpoints we have? No, apparently this not meant here. 
Perhaps `promregator.endpoint.single.maxTargets`?